LDKNodeMonday/Extensions/String+Extensions.swift

```diff
@@ -220,4 +206,11 @@ extension String {
         return self.uppercased()
     }
 
+    func truncateMiddle(first: Int, last: Int) -> String {
+        guard self.count > first + last else { return self }
+        let start = self.prefix(first)
+        let end = self.suffix(last)
+        return "\(start)…\(end)"
+    }
+
```

Comment:
There might be a reason this custom function is needed (if so leave it), i looked at the call site and couldn’t tell totally for sure, but if we can use https://developer.apple.com/documentation/swiftui/view/truncationmode(_:) instead that would be preferred I’m thinking.

Reply:
truncationmode() doesn't allow for specifying how many remaining numbers you want, so it ends up showing a very long string. So I ended up adding this.